test/test-condvar.c

```diff
@@ -242,25 +237,11 @@ TEST_IMPL(condvar_5) {
 
   uv_mutex_lock(&wc.mutex);
 
-  /* We wait.
-   * No signaler, so this will only return if timeout is delivered. */
-  before = uv_hrtime();
-  r = uv_cond_timedwait(&wc.cond, &wc.mutex, timeout);
-  after = uv_hrtime();
+  /* We wait. No signaler, so this will only return if timeout is delivered. */
+  ASSERT_EQ(UV_ETIMEDOUT, uv_cond_timedwait(&wc.cond, &wc.mutex, timeout));
 
   uv_mutex_unlock(&wc.mutex);
 
-  /* It timed out. */
-  ASSERT(r == UV_ETIMEDOUT);
-
-  /* It must have taken at least timeout, modulo system timer ticks.
-   * But it should not take too much longer.
-   * cf. MSDN docs:
-   * https://msdn.microsoft.com/en-us/library/ms687069(VS.85).aspx */
```

Comment:
> If the time-out interval is less than the resolution of the system clock, the wait may time out in less than the specified length of time. If the time-out interval is greater than one tick but less than two, the wait can be anywhere between one and two ticks, and so on

FWIW, this claim in the documentation is false. Microsoft confirmed that it is incorrect, and I suggested they delete this text, but they did not (yet): https://github.com/MicrosoftDocs/sdk-api/pull/526